### PR TITLE
#6 README - note about XText dialogue

### PR DESCRIPTION
**Pull request by paulvi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,6 +20,13 @@ http://marketplace.eclipse.org/content/jbc/
 
 ## Usage
 
+When first time opening Editor within a project, a dialogue will appear with suggestion to convert to XText project.
```
Comment (kthoms) [inline README.md]: The correct notation is "Xtext"
See https://blogs.itemis.com/en/about-the-notation-of-the-names-xtext-xtend-xbase-xcore-xpand

File: README.md [hunk 2]
```diff
@@ -20,6 +20,13 @@ http://marketplace.eclipse.org/content/jbc/
 
 ## Usage
 
+When first time opening Editor within a project, a dialogue will appear with suggestion to convert to XText project.  
+Saying "No" is perfectly ok, the editor does not require the nature.  
+Probably in a future xtext version one can introduce a method in the XtextEditor to opt-out from the dialog.
```
Comment (kthoms) [inline README.md]: Also here, not "xtext", but "Xtext"

**Review by kthoms — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (paulvi) [reply README.md]: I know, I just copied from #6

**Comment by paulvi** [comment]
updated to Xtext

**Comment by paulvi** [comment]
Frankly for such smaller change it is better to accept, and than change letters directly

**Comment by kthoms** [comment]
I wanted to make you aware of it. It is written too often wrong.

**Review by kthoms — APPROVED** [review]

**Comment by kthoms** [comment]
Thanks for the update!

**Pull request merged**